husky_components/data-navigation/main.js

```diff
@@ -338,17 +338,18 @@ define([
             if (!!this.data.hasNextPage && !this.loading) {
                 this.showLoader();
 
-                this.page++;
-                this.sandbox.util.load(this.getUrl(this.getCurrentUrl()))
-                    .then(function(data) {
-                        var children = data._embedded[this.options.resultKey] || [];
-                        this.data.children = this.data.children.concat(children);
-                        this.data.hasNextPage = this.page < data.pages;
-                        this.currentView.append(children, this.options);
-
-                        this.hideLoader();
-                        def.resolve();
-                    }.bind(this));
+                this.page += 1;
```

Comment:
It's just a matter of taste. I'm not a fan of "++" and "--" as they have side-effects. In my opinion "+=" makes makes it more clear that there's an assignment happening.